files/matreshka/update.json

```diff
@@ -3,6 +3,13 @@
   "name": "matreshka",
   "repo": "matreshkajs/matreshka",
   "files": {
-    "include": ["matreshka.min.js", "matreshka.js", "matreshka.min.map"]
+    "include": [
+		"matreshka.min.js",
+		"matreshka.js",
+		"matreshka.min.map",
+		"magic/matreshka-magic.min.js",
```

Comment:
You can use the Github web interface for this kind of changes. Its very easy and fast